.gitattributes

```diff
@@ -1,14 +1,4 @@
-# @TODO 3.0 replace following `tests/... export-ignore` with single `tests/ export-ignore`
-/tests/**/*Test.php export-ignore
-/tests/AbstractDoctrineAnnotationFixerTestCase.php export-ignore
-/tests/Console/TestToolInfo.php export-ignore
-/tests/Differ/AbstractDifferTestCase.php export-ignore
-/tests/Differ/DummyTestSplFileInfo.php export-ignore
-/tests/Fixtures/ export-ignore
-/tests/Linter/AbstractLinterTestCase.php export-ignore
-/tests/Report/AbstractReporterTestCase.php export-ignore
-/tests/Test/AbstractFixerWithAliasedOptionsTestCase.php export-ignore
-/tests/Test/AbstractTransformerTestCase.php export-ignore
+/tests/ export-ignore
```

Comment:
Let's put the line in the proper place within the list below.

Reply:
done